doc/sphinx/addendum/implicit-coercions.rst

```diff
@@ -37,12 +37,13 @@ In addition to these user-defined classes, we have two built-in classes:
   * ``Funclass``, the class of functions; its objects are all the terms with a functional
     type, i.e. of form :g:`forall x:A,B`.
 
-Formally, the syntax of classes is defined as:
+   .. insertprodn class class
+
+   .. prodn::
+      class ::= Funclass
+      | Sortclass
+      | @smart_global
```

Comment:
I am thinking of renaming `smart_global` to `smart_qualid` to better show that this is just a qualid that may be designated by an associated notation. And it should not stay defined where it is today. At some point, its definition should be moved to wherever notations are introduced.

Reply:
Thinking or decided?

Reply:
That's a proposal that I submit to your approval.

Reply:
It makes sense.  I renamed it.